src/meta/Drag.ts

```diff
@@ -153,9 +153,12 @@ class DragController {
 		if (this._nodeMap.has(target)) {
 			return { state: this._nodeMap.get(target)!, target };
 		}
-		if (target.parentElement) {
-			return this._getData(target.parentElement);
-		}
+		// IE 11 does not support parentElement for SVGElement, so in case of parentElement undefined
+        // checking for parentNode.
```

Comment:
Need to set your editor's linting rules to match what Dojo provides... need tabs instead of spaces for this PR please. Otherwise this change looks fine to me!

Reply:
Closing this PR because of author email address is my wrong in commit. I will raise another PR with above changes.